<launch>
  <arg name="maxFeatures" default="10"/>
  <arg name="bucketHeight" default="50"/>
  <arg name="bucketWidth" default="50"/>
  <arg name="subPixel" default="2"/>
  <arg name="f" default="850"/>
  <arg name="cu" default="550"/>
  <arg name="cv" default="444"/>
  <arg name="baseline" default="0.120"/>
  <arg name="outputTopic" default="viso_config"/>
  <param name="$(arg outputTopic)/maxFeatures" type="int" value="$(arg maxFeatures)"/>
  <param name="$(arg outputTopic)/bucketHeight" type="int" value="$(arg bucketHeight)"/>
  <param name="$(arg outputTopic)/bucketWidth" type="int" value="$(arg bucketWidth)"/>
  <param name="$(arg outputTopic)/subPixel" type="int" value="$(arg subPixel)"/>
  <param name="$(arg outputTopic)/f" type="double" value="$(arg f)"/>
  <param name="$(arg outputTopic)/cu" type="double" value="$(arg cu)"/>
  <param name="$(arg outputTopic)/cv" type="double" value="$(arg cv)"/>
  <param name="$(arg outputTopic)/baseline" type="double" value="$(arg baseline)"/>
  <node name="viso_extractor" pkg="viso_extractor" type="viso_node" output="screen"/>
</launch>
